### PR TITLE
TDL-6141: Switch from using multipleOf to singer.decimal

### PR DESCRIPTION
**Pull request by savan-chovatiya**
# Description of change
TDL-6141: Switch from using multipleOf to singer.decimal

- Changed from using type number and multipleOf 1e-20 to type string and format singer.decimal in funnels and revenue streams

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tap_mixpanel/schemas/funnels.json [hunk 1]
```diff
@@ -3,60 +3,105 @@
...
-      "type": ["null", "string"],
+      "type": [
+        "null",
+        "string"
+      ],
       "format": "date"
     },
     "datetime": {
-      "type": ["null", "string"],
+      "type": [
```
Comment (karanpanchal-crest) [inline tap_mixpanel/schemas/funnels.json:25]: @namrata270998 Re-structuring the "type" array is not expected as part of this PR. Can you please revert this change?

**Review by karanpanchal-crest — APPROVED** [review]

**Comment by prijendev** [comment]
The code change of this PR is already merged in the main branch. Hence, closing this PR.
